package.json

```diff
@@ -42,6 +42,7 @@
     "bignumber.js": "8.0.1",
     "bitcoinjs-message": "2.0.0",
     "bitgo-utxo-lib": "1.3.1",
+    "@bitgo/unspents": "0.5.0",
```

Comment:
Should be at the top (these are sorted alphabetically, including special characters)

Reply:
Done